plugins/modules/aci_access_span_filter_group_entry.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Akini Ross <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_access_span_filter_group_entry
+short_description: Manage Access SPAN filter group entries (span:FilterEntry)
+description:
+- Manage Access SPAN filter group entries on Cisco ACI fabrics.
+options:
+  filter_group:
+    description:
+    - The name of the Access SPAN filter group.
+    type: str
+  source_ip:
+    description:
+    - The source IP Prefix.
+    type: str
+  destination_ip:
+    description:
+    - The destination IP Prefix.
+    type: str
+  first_src_port:
+    description:
+    - The first source port (from port).
+    - The APIC defaults to C(unspecified) when unset during creation.
+    type: str
+    choices: [ unspecified, dns, ftp, http, https, pop3, rtsp, smtp, ssh ]
```

Comment:
Are those the only choices or can you also send any valid port number?

Reply:
You can send port numbers, which are the translated into string equivalent by APIC. In UI and the GET requests the port number will not be visible. Do you want to support input for port numbers or match UI?